### ISSUE TITLE
[Bug]: [Accessibility]  No focus on icon button  with Talkback

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#393]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung A53

### OS Version

Android 11

### App version

1.0

### Describe the issue

Go to Components -> button -> Icon buttons 
Activate Talkback
No Focus on icon button with Talkback 

### Expected Behavior

Add focus on icon button
